quickwit/quickwit-cli/src/index.rs

```diff
@@ -598,11 +604,7 @@ pub async fn create_index_cli(args: CreateIndexArgs) -> anyhow::Result<()> {
         }
     }
 
-    let index_service = IndexService::new(
-        metastore,
-        quickwit_storage_uri_resolver().clone(),
-        quickwit_config.default_index_root_uri,
```

Comment:
We lose the default_index_root_uri because resolving the index uri is now part of deserialization.